Cargo.toml

```diff
@@ -189,6 +189,8 @@ snapbox = "0.6.16"
 shlex = "1.3.0"
```

Comment:
Just adding a new example to demonstrate, and some dev-deps to support the example.